Configuration/PyReleaseValidation/python/MatrixInjector.py

```diff
@@ -62,7 +62,7 @@ def __init__(self,opt,mode='init',options=''):
                 self.DbsUrl = "https://"+self.wmagent+"/dbs/int/global/DBSReader"
 
         if not self.dqmgui:
-            self.dqmgui="https://cmsweb.cern.ch/dqm/relval;https://cmsweb-testbed.cern.ch/dqm/relval"
+            self.dqmgui="https://cmsweb.cern.ch/dqm/relval"
```

Comment:
The test DQM gui url is stopping service on Oct. 12, and we have been
requested to remove it from all 74x 75x and 76x.   PRs to 75x and 76x will
come soon.  Giovanni @franzoni , could help to confirm our plan.

Cheers, Hengne

On Mon, Sep 28, 2015 at 5:56 PM, Chris Jones <notifications@github.com>
wrote:

> In Configuration/PyReleaseValidation/python/MatrixInjector.py
> <https://github.com/cms-sw/cmssw/pull/11424#discussion_r40569186>:
>
> > @@ -62,7 +62,7 @@ def __init__(self,opt,mode='init',options=''):
> >                  self.DbsUrl = "https://"+self.wmagent+"/dbs/int/global/DBSReader"
> >
> >          if not self.dqmgui:
> > -            self.dqmgui="https://cmsweb.cern.ch/dqm/relval;https://cmsweb-testbed.cern.ch/dqm/relval"
> > +            self.dqmgui="https://cmsweb.cern.ch/dqm/relval"
>
> Why was the other URL removed? Looking in CMSSW_7_6_2015-09-27-2300 that
> URL is still in that release.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/11424/files#r40569186>.
>